odoo-openmrs/src/main/java/com/ozonehis/eip/odoo/openmrs/handlers/SaleOrderHandler.java

```diff
@@ -123,7 +122,9 @@ public void updateSaleOrderIfExistsWithSaleOrderLine(
         }
 
         // Update sale order with Patient Weight if not already present
-        if (saleOrder.getPartnerWeight() == null || saleOrder.getPartnerWeight().isEmpty()) {
+        if (saleOrder.getPartnerWeight() == null
+                || saleOrder.getPartnerWeight().isEmpty()
+                || saleOrder.getPartnerWeight().equals("false")) {
```

Comment:
Is this not supposed to get the actual numeric weight of the partner instead of a boolean?

Reply:
So in Odoo if the field is null it is set as `False` that's why we cast it to `String` in our eip routes.